Analysis/include/EventInfo.h

```diff
@@ -363,7 +447,27 @@ class EventInfoBase {
         return *fatJets;
     }
 
-    bool HasBjetPair() const { return has_bjet_pair; }
+    bool HasBjetPair() const { return selected_signal_jets.HasBjetPair(GetNJets()); }
+
+    bool HasVBFjetPair() const { return selected_signal_jets.HasVBFPair(GetNJets()); }
+
+    const JetCandidate& GetVBFJet(const size_t index)
+    {
+        if(index != 1 || index != 2 || !HasVBFjetPair())
```

Comment:
condition is wrong. should be:
if(!HasVBFJetPair() || (index != 1 && index != 2))